### PR TITLE
[doc] Improve tls authentication documentation.

### PR DESCRIPTION
**Pull request by codelipenghui**
### Motivation

According to current documentation, for a beginner can't finish each step smoothly. So add some keys that might go wrong at these steps.

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: site2/docs/security-tls-authentication.md [hunk 1]
```diff
@@ -26,9 +26,10 @@ $ openssl pkcs8 -topk8 -inform PEM -outform PEM \
       -in admin.key.pem -out admin.key-pk8.pem -nocrypt
 ```
 
-Generate the certificate request. When asked for a **common name**, enter the **role token** which you want this key pair to authenticate a client as.
+Generate the certificate request. When asked for a **common name**, enter the **role token** which you want this key pair to authenticate a client as. 
 
 ```bash
+$ cp /etc/pki/tls/openssl.cnf ./
```
Comment (sijie) [inline site2/docs/security-tls-authentication.md]: hmm I think the documentation here was split from http://pulsar.apache.org/docs/en/security-tls-authentication/ . Hence there is an assumption that `openssl.cnf` was fetched in http://pulsar.apache.org/docs/en/security-tls-transport/#certificate-authority 

So I am not sure the fix is correct.

**Comment by jiazhai** [comment]
@codelipenghui Would you please help check @sijie 's comments?

**Comment by codelipenghui** [comment]
@sijie @jiazhai I'm already updated the document, PTAL.

**Comment by codelipenghui** [comment]
run Integration Tests
run Java8 Tests

**Comment by codelipenghui** [comment]
run Integration Tests

**Pull request merged**